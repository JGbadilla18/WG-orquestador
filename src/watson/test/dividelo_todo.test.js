const testWatson = require('./testWatson');

watsonText = 'dividelo todo'

it(watsonText, () => {
  expect.assertions(1);
  return testWatson(watsonText).then(data => expect(data).toMatchSnapshot());
});