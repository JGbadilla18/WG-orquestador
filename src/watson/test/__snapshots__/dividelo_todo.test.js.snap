// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`dividelo todo 1`] = `
Object {
  "actions": Array [],
  "components": Array [
    Object {
      "elements": Array [
        Object {
          "buttons": Array [
            Object {
              "title": "Conoce más aquí",
              "type": "web_url",
              "url": "https://blog.corporacionbi.com/tarjeta-credito/productos/dividelo-todo",
            },
          ],
          "image_url": "http://www.bi-social.com/clubbi/wp-content/uploads/2020/05/Dividelo-Todo.jpg",
          "subtitle": "Plan de financiamiento para tarjetas de crédito donde puedes comprar en cuotas.",
          "title": "Divídelo Todo",
        },
        Object {
          "buttons": Array [
            Object {
              "title": "Conoce más aquí",
              "type": "web_url",
              "url": "https://blog.corporacionbi.com/tarjeta-credito/productos/dividelo-todo/categorias",
            },
          ],
          "image_url": "http://www.bi-social.com/clubbi/wp-content/uploads/2020/05/Promociones-Dividelo-Todo.jpg",
          "subtitle": "¡Descubre promociones, descuentos y beneficios al comprar con Divídelo Todo!",
          "title": "Ingresa a la categoría de tu interés",
        },
      ],
      "response_index": 0,
      "response_type": "carousel",
    },
    Object {
      "response_type": "text",
      "text": "Solicita Divídelo Todo con tus Tarjetas de Crédito de Banco Industrial para comprar en cuotas en tu establecimiento favorito sin ningun recargo de interés.

  *  Compras en cuotas desde Q200 (aplica según políticas de comercio)
  *  Plazos de 2, 3, 6, 10, 12, 15, 18 y 24 (aplica según políticas de comercio)
  *  Sin recargo de interés
  *  Acumulas Bi Puntos",
    },
    Object {
      "response_type": "pause",
      "time": 10000,
      "typing": true,
    },
  ],
}
`;
